src/screens/Onboarding/StepFinished.tsx

```diff
@@ -185,6 +187,7 @@ export function StepFinished() {
     setActiveStarterPack(undefined)
     setHasCheckedForStarterPack(true)
     setQueuedTour(TOURS.HOME)
+    startProgressGuide('like-10-and-follow-7')
```

Comment:
Should we check the gate here in case we want to test entirely separate variations or do you want to keep it more centralized?